app/api/evidences_vault/specs/vaultSync.spec.js

```diff
@@ -105,22 +105,27 @@ describe('vaultSync', () => {
       expect(await getFileContent('1.png')).toBe('this is a fake image');
       expect(await getFileContent('2.png')).toBe('this is a fake image');
 
-      expect(imported.map(e => e.metadata.screenshot[0].value)).toEqual([
-        `/api/attachments/download?_id=${imported[0]._id}&file=1.png`,
-        `/api/attachments/download?_id=${imported[1]._id}&file=2.png`,
-      ]);
-
-      expect(imported[0].attachments.find(a => a.filename.match(/png/))).toEqual(
+      const firstPngAttachment = imported[0].attachments.find(a => a.filename.match(/png/));
+      expect(firstPngAttachment).toEqual(
         expect.objectContaining({
           filename: '1.png',
         })
       );
 
-      expect(imported[1].attachments.find(a => a.filename.match(/png/))).toEqual(
+      expect(imported[0].metadata.screenshot[0]).toEqual({
+        value: 'api/files/1.png',
```

Comment:
@konz the values for all evidente vault metadata that are urls need to have a leading "/" like in the test before the change, otherwise the videos and images do not show up in uwazi ui. everything else seems to work properly !👍

Reply:
fixed

Reply:
fixed